definitions/ext-service/definition.yml

```diff
@@ -28,6 +28,20 @@ synthesis:
           entityTagName: language
         telemetry.sdk.version:
 
+    # telemetry from JFR uploader
+    - identifier: service.name
```

Comment:
nit but should we put this near the other JFR-UPLOADER rule?

Reply:
oh there is one already!

Reply:
yeah but is by `instrumentation.provider` would that match the same entities as this one?

Reply:
Yeah, it was enough with instrumenation.provider.  The issue was that we had disabled EXT-SERVICE in the ingestion pipeline that processed the JFR-Uploader events.